EIPS/eip-7668.md

```diff
@@ -12,23 +12,23 @@ created: 2024-03-31
 
 ## Abstract
 
-Require the bloom filters in an execution block, including at the top level and in the receipt object, to be empty.
+Require the bloom filters in an execution block, including at the top level and in the receipt object, to be hardcoded to `0xffff...ff`.
 
 ## Motivation
 
 Logs were originally introduced to give applications a way to record information about onchain events, which decentralized applications (dapps) would be able to easily query. Using bloom filters, dapps would be able to quickly go through the history, identify the few blocks that contained logs relative to their application, and then quickly identify which individual transactions have the logs that they need.
 
-In practice, this mechanism is far too slow. Almost all dapps that access history end up doing so not through RPC calls to an Ethereum node (even a remote-hosted one), but through centralized extra-protocol services.
+In practice, this mechanism is far too slow. Almost all dapps that access history end up doing so not through RPC calls to an Ethereum node (even a remote-hosted one), but through centralized extra-protocol services.  Not too mention, in a chain with relatively full blocks, the false positive rate in bloom filters has increased to the point of making bloom filters almost useless.
 
-This EIP proposes recognizing that reality, and removing bloom filters from the protocol. Applications that need history querying would be encouraged to develop and use decentralized protocols that create provable log indexes using eg. ZK-SNARKs and incrementally-verifiable computation.
+This EIP proposes recognizing that reality, and removing bloom filters from the protocol since it places an unnecessary compute burden on execution clients computing what is for all prati. Applications that need history querying would be encouraged to develop and use decentralized protocols that create provable log indexes using eg. ZK-SNARKs and incrementally-verifiable computation.
 
 ## Specification
 
-The logs bloom of an execution block is now required to be empty (ie. 0 bytes long). The logs bloom of a transaction receipt is now required to be empty (ie. 0 bytes long).
+The `logsbloom` field of the execution block header and the `logsbloom` field of the transaction receipt is now required to be `0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff` (ie. 32 bytes filled with `ff`).
 
 ## Rationale
 
-This is a minimally disruptive way to remove the need to handle blooms from clients. A future EIP can later clean up by removing this field entirely, along with other fields that have been deprecated.
+This is a minimally disruptive way to remove the need to compute blooms from clients while not requiring any breaking consensus changes.
```

Comment:
Well, this is a breaking consensus change, since it forces the bloom filter to be a constant which in almost all practical cases will not be the produced bloom filter when executing the block.

Reply:
Well, breaking on the level of "we're changing how this field is computed" but breaking the structure of the values that are RLP encoded into the serialized form of the header and receipt, right?